### PR TITLE
Adding IP_TTL socket option while restoring

### PR DESCRIPTION
**Pull request by rahulk789**
Regarding issue #2007

### CODE REVIEW

File: test/zdtm/static/sock_ip_opts00.c [hunk 1]
```diff
@@ -26,6 +26,7 @@ struct sk_opt {
 struct sk_opt sk_opts_v4[] = {
 	{ SOL_IP, IP_FREEBIND, IP_OPT_VAL },
 	{ SOL_IP, IP_PKTINFO, IP_OPT_VAL },
+	{ SOL_IP, IP_TTL, IP_OPT_VAL },
```
Comment (rst0git) [inline test/zdtm/static/sock_ip_opts00.c:29]: `val` should be a value between 1 and 255, not `IP_OPT_VAL`.
You can run the test locally with `sudo python3 zdtm.py run -t zdtm/static/sock_ip_opts01`

Comment (rahulk789) [reply test/zdtm/static/sock_ip_opts00.c]: I have ran the tests locally. Changed the value to 32 .

File: test/zdtm/static/sock_ip_opts00.c [hunk 2]
```diff
@@ -26,7 +26,7 @@ struct sk_opt {
 struct sk_opt sk_opts_v4[] = {
 	{ SOL_IP, IP_FREEBIND, IP_OPT_VAL },
 	{ SOL_IP, IP_PKTINFO, IP_OPT_VAL },
-	{ SOL_IP, IP_TTL, IP_OPT_VAL },
+	{ SOL_IP, IP_TTL, 32 },
```
Comment (rst0git) [inline test/zdtm/static/sock_ip_opts00.c:29]: @rahulk789 would you be able to squash this change in the previous commit (`git rebase -i HEAD~2`)?

Comment (rahulk789) [reply test/zdtm/static/sock_ip_opts00.c:29]: Yup, was a bit busy with work, will do so. Thanks.

**Comment by rahulk789** [comment]
Any updates on this?

**Pull request merged**